extensions/Worker.Extensions.Tables/src/TypeConverters/TableData.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Tables.TypeConverters
+{
+    internal class TableData
```

Comment:
is this used outside the `TableConverterBase` at all? If not, consider moving it to be a nested `protected` class of that type.